am-delete-aip/ss-delete-dip.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Delete a DIP from the Storage Service database.
+Use the python interpreter of the storage service virtualenv.
+Run with user archivematica.
+"""
+
+from __future__ import print_function
+import argparse
+import getpass
+import os
+import sys
+
+user = getpass.getuser()
+if (user != 'archivematica'):
+    print("Error: user is {}, run script with user archivematica (sudo -u archivematica)".format(user))
+    sys.exit()
+
+sys.path.append('/usr/lib/archivematica/storage-service')
+os.environ['DJANGO_SETTINGS_MODULE'] = 'storage_service.settings.production'
+os.environ['DJANGO_SECRET_KEY'] = 'dummy'
+os.environ['SS_DB_NAME'] = '/var/archivematica/storage-service/storage.db'
+os.environ['SS_DB_USER'] = 'dummy'
+os.environ['SS_DB_PASSWORD'] = 'dummy'
+os.environ['SS_DB_HOST'] = 'dummy'
+
+import django
+django.setup()
```

Comment:
You don't need to do this after the import anymore. I think it looks cleaner when done from __main__ or main().